FirebasePerformance/Tests/Unit/Instruments/FIRHTTPMetricTests.m

```diff
@@ -187,7 +185,6 @@ - (void)testMetricCreationFailsWhenResponseCodeNotSet {
   [metric markResponseStart];
   [metric stop];
   FPRNetworkTrace *networkTrace = metric.networkTrace;
-  XCTAssertNil(FPRGetNetworkRequestMetric(networkTrace));
```

Comment:
Why are these removed? These are valid tests, isn't it?

Reply:
Because it returns a struct instead of a pointer now. I updated the test to check if the networkTrace is valid. Please take a look.